README.md

```diff
@@ -43,6 +43,11 @@ Screen recording of AFL running on Rust code.
 
 ## Hints
 
+Before starting to fuzz, you should reconfigure your system for optimal
+performance and better crash detection. This can be done with `cargo-afl afl system-config`.
+But this subcommand requires root, so it uses sudo internally. Hence, you will need to enter
```

Comment:
```suggestion
But this subcommand requires root, so it uses sudo internally. Hence, you might need to enter
```
To make this consistent with the help message below. (Sorry for going back and forth.)